config/Run3_2022/samples.yaml

```diff
@@ -42,3 +36,1092 @@ MET_Run2022C:
+  sampleType: W
+
+WtoLNu_1J_madgraphMLM:
+  crossSection: WtoLNu_1J_madgraphMLM
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_1J_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v1/MINIAODSIM
+  sampleType: W
+WtoLNu_2J_madgraphMLM:
+  crossSection: WtoLNu_2J_madgraphMLM
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_2J_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v1/MINIAODSIM
+  sampleType: W
+WtoLNu_3J_madgraphMLM:
+  crossSection: WtoLNu_3J_madgraphMLM
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_3J_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: W
+WtoLNu_4J_madgraphMLM:
+  crossSection: WtoLNu_4J_madgraphMLM
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_4J_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: W
+
+WtoLNu_MLNu-0to120_HT-40To100_madgraph:
+  crossSection: WtoLNu_MLNu-0to120_HT-40To100_madgraph
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_MLNu-0to120_HT-40to100_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v3/MINIAODSIM
+  sampleType: W
+WtoLNu_MLNu-0to120_HT-100To400_madgraph:
+  crossSection: WtoLNu_MLNu-0to120_HT-100To400_madgraph
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_MLNu-0to120_HT-100to400_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v3/MINIAODSIM
+  sampleType: W
+WtoLNu_MLNu-0to120_HT-400To800_madgraph:
+  crossSection: WtoLNu_MLNu-0to120_HT-400To800_madgraph
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_MLNu-0to120_HT-400to800_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v1/MINIAODSIM
+  sampleType: W
+WtoLNu_MLNu-0to120_HT-800To1500_madgraph:
+  crossSection: WtoLNu_MLNu-0to120_HT-800To1500_madgraph
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_MLNu-0to120_HT-800to1500_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: W
+WtoLNu_MLNu-0to120_HT-1500To2500_madgraph:
+  crossSection: WtoLNu_MLNu-0to120_HT-1500To2500_madgraph
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_MLNu-0to120_HT-1500to2500_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: W
+
+WtoLNu_amcatnloFXFX:
+  crossSection: WtoLNu
+  generator: amcatnloFXFX
+  miniAOD: /WtoLNu-2Jets_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: W
+
+WtoLNu_madgraphMLM:
+  crossSection: WtoLNu
+  generator: madgraphMLM
+  miniAOD: /WtoLNu-4Jets_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: W
+WtoLNu_madgraphMLM_ext1:
+  crossSection: WtoLNu_madgraphMLM
```

Comment:
`WtoLNu_madgraphMLM` is not defined

Reply:
Fixed, it was an extension sample that should have matched with the WtoLNu (which is also now changed to WtoLNu_NNLO+NLO blah blah blah